A3A/addons/core/functions/init/fn_initServer.sqf

```diff
@@ -50,6 +50,8 @@ Info("Background init started");
 
 // No reason not to do this early
 [] execVM QPATHTOFOLDER(Scripts\fn_advancedTowingInit.sqf);
+[] execVM QPATHTOFOLDER(Scripts\SpectumDevice\spectrum_device.sqf);
+[] execVM QPATHTOFOLDER(Scripts\SpectumDevice\sa_ewar.sqf);
```

Comment:
Does the server really need this too?

Reply:
Not sure, I just copied what advanced towing had, so... probably?